src/dotnet-interactive/dotnet-interactive.csproj

```diff
@@ -36,6 +36,14 @@
     <None Remove="resources\*.js" />
   </ItemGroup>
 
+  <PropertyGroup>
+    <MicrosoftCodeAnalysisAnalyzersVersion>2.9.6</MicrosoftCodeAnalysisAnalyzersVersion>
+    <SystemRuntimeLoaderVersion>4.3.0</SystemRuntimeLoaderVersion>
+    <SystemCollectionsImmutableVersion>5.0.0</SystemCollectionsImmutableVersion>
+    <SystemReflectionMetadataVersion>5.0.0</SystemReflectionMetadataVersion>
+    <SystemCommandLineVersion>2.0.0-beta1.21308.1</SystemCommandLineVersion>
+  </PropertyGroup>
```

Comment:
Where did all these values come from and where are they used?  The Arcade way of setting package values like this is to put them at the end of `eng/Versions.props` to keep them collected.